omero/users/clients-overview.txt

```diff
@@ -56,7 +56,12 @@ views.
     :alt: OMERO.web image viewer
     
     OMERO.web image viewer
-    
+
+OMERO.web now features almost all of the functionality of OMERO.insight
+including the ability to administrate users and groups. Work is ongoing to
```

Comment:
Yes but it wasn't mentioned here whereas there's a whole list of stuff insight includes. Sorry confusing wording I guess, I'll do something with it tomorrow